src/modelmanager.cpp

```diff
@@ -366,9 +366,9 @@ static Status processCustomNodeConfig(const rapidjson::Value& nodeConfig, Custom
     return StatusCode::OK;
 }
 
-static Status processMediapipeConfig(rapidjson::Document& configJson, const rapidjson::Value& pipelineConfig, std::set<std::string>& mediapipesInConfigFile, MediapipeFactory& factory, ModelManager& manager) {
+Status ModelManager::processMediapipeConfig(rapidjson::Document& configJson, const rapidjson::Value& pipelineConfig, std::set<std::string>& mediapipesInConfigFile, MediapipeFactory& factory, std::vector<ModelConfig>& gatedModelConfigs) {
```

Comment:
Why we need to make this method instead of function? Idea here is that eventually we should separate configuration parsing logic from ModelManager so at least this function was ready.